pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
@jeremybmerrill this is the cause of the failures and my analysis of it. I don't know Travis super well, so I'm not sure how to fix it.